support-frontend/assets/helpers/redux/checkout/address/validation.ts

```diff
@@ -165,6 +169,20 @@ function getDeliveryOnlyRules(
 				'The address and postcode you entered is outside of our delivery area. Please go back to purchase a voucher subscription instead.',
 			),
 		},
+		{
+			rule:
+				abParticipations.nationalDelivery === 'variant'
+					? isSaturdayOrSundayDeliveryAvailable(
+							fulfilmentOption,
+							fields.postCode,
+							productOption,
+					  )
+					: true,
+			error: formError(
+				'postCode',
+				'Saturday or Sunday delivery is available for Greater London only',
```

Comment:
Should this direct users back? I’m checking with Rob

```suggestion
				'Saturday or Sunday delivery is available for Greater London only. Please go back to purchase weekend delivery instead.',
```

Reply:
Got this suggestion from Rob:

```suggestion
				'Saturday or Sunday delivery is available for Greater London only. Go back and select Weekend delivery or choose a Digital Voucher.',
```